sdks/java/io/hadoop-input-format/hifio/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java

```diff
@@ -845,7 +845,8 @@ public Double getFractionConsumed() {
        */
```

Comment:
also, https://github.com/diptikul/incubator-beam/blob/9f69f264f42c6e3cda3690a6e4d2ce9f298da4b7/sdks/java/io/hadoop-input-format/hifio/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java#L347 - we use VisibleForTesting on a public method, which shouldn't be necessary.